src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
+	 * @since TBD
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 1d021d76a4bce854a53ce5eb10356a56</sub>